### PR TITLE
bokeh 3.2.0 foss/2022b

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/8612a4d764b42ec9ce8d65ae4b81a9fb for a full test report.

File: easybuild/easyconfigs/b/bokeh/bokeh-3.2.0-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        'checksums': ['d8165a088d31798b59e91117d1f5fc3df8168d8b48c4acc10fc0df0d0bdbcc5e'],
+    }),
+    ('xyzservices', '2023.7.0', {
+        'checksums': ['0ec928742227d6f5d4367ea7b457fcfed943429f4de2949b5b02a82cdf5569d6'],
+    }),
+    ('setuptools', '67.8.0', {
+        'checksums': ['62642358adc77ffa87233bc4d2354c4b2682d214048f500964dbe760ccedf102'],
+    }),
+    (name, version, {
+        'preinstallopts': """sed -i 's/setup(/setup(version="%(version)s",/g' setup.py && """,
```
Comment (casparvl) [inline easybuild/easyconfigs/b/bokeh/bokeh-3.2.0-foss-2022b.eb:38]: This looks to be new and clearly fixes something in the sources. Can you include a comment what was wrong here that needed to be fixed?

File: easybuild/easyconfigs/b/bokeh/bokeh-3.2.0-foss-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+name = 'bokeh'
+version = '3.2.0'
+
+homepage = 'https://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
```
Comment (casparvl) [inline easybuild/easyconfigs/b/bokeh/bokeh-3.2.0-foss-2022b.eb:11]: If there are no build dependencies, this can be ommitted.

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by casparvl** [comment]
@fizwit [this](https://github.com/easybuilders/easybuild-easyconfigs/pull/18470) PR came in and has _a bit_ newer version. Would this also work for you? Also, I see here that `setuptools` and `meson-python` were put in the build dependencies there, which might make more sense?

**Comment by casparvl** [comment]
Had a small discussion with fellow maintainers on Slack. Though we have definitely _not_ been consistent in the past, there seems to be a preference to using `setuptools` from a seperate EasyConfig as a build dependency.

Wondering how to move forward with this PR, also up to you @fizwit . If you care strongly about this particular version, and the one from [here](https://github.com/easybuilders/easybuild-easyconfigs/pull/18470) doesn't work for you, we can proceed with this PR. At the very least, I think `meson-python` can be moved to build dependencies, right? Then you can ignore my earlier comment in my review. I'd be ok in merging it like that, even if it doesn't use `setuptools` as external dependency.

**Comment by fizwit** [comment]
Closing PR, the work is duplicated in PR #18470

**Pull request closed without merging**